test/node_modules/side-effects-array/package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"main": "./index.js",
 	"sideEffects": [
+		"./index.js",
 		"./dep1.js",
 		"**/*-free.js"
```

Comment:
One last note, maybe to make it less confusing, we could rename the `-free` files to e.g. `-effect`?